src/run.test.ts

```diff
@@ -86,7 +86,13 @@ describe('run.ts', () => {
       expect(os.type).toHaveBeenCalled()
    })
 
-   test('getLatestHelmVersion() - return the stable version of HELM since its not authenticated', async () => {
+   test('getLatestHelmVersion() - return the latest version of HELM', async () => {
+      expect(await run.getLatestHelmVersion()).toBe('v3.14.2')
```

Comment:
can you update this test to use a jest mock? This will become outdated when they release a new version. We shouldn't have to constantly update the version here in the test.